geode-redis/src/main/java/org/apache/geode/redis/internal/ExecutionHandlerContext.java

```diff
@@ -146,6 +146,7 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
       executeCommand(ctx, command);
     } catch (Exception e) {
       logger.error(e);
+      e.printStackTrace();
```

Comment:
Remove this printStackTrace. logger.error will log a stack trace if you do this: logger.error("description of context", e);
Basically we never want to see e.printStackTrace in geode code. Always use a logger instead.

Reply:
done - thanks!